src/Licenses/Entities/Offer/PriceBand/Billing.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace ArrowSphere\PublicApiClient\Licenses\Entities\Offer\PriceBand;
+
+use ArrowSphere\PublicApiClient\AbstractEntity;
+
+/**
+ * Class Billing
+ */
+class Billing extends AbstractEntity
+{
+    public const COLUMN_CYCLE = 'cycle';
+
+    public const COLUMN_TERM = 'term';
+
+    public const COLUMN_TYPE = 'type';
+
+    protected const VALIDATION_RULES = [
```

Comment:
The validation rules are missing. According to the constructor, all 3 fields are mandatory.